ZuluContent/Zulu/Items/Resources/Lumberjack/BaseLog.cs

```diff
@@ -0,0 +1,56 @@
+namespace Server.Items
+{
+    [FlipableAttribute(0x1bdd, 0x1be0)]
+    public class BaseLog : Item
+    {
+        [CommandProperty(AccessLevel.GameMaster)]
+        public CraftResource Resource { get; set; }
+
+        public override string DefaultName => $"{CraftResources.GetName(Resource)}";
```

Comment:
Not sure why string interpolation is here, should just be `=> CraftResources.GetName(Resource)`

Reply:
ah yeah it had "ore" in it when I copied it over from BaseOre and I forgot to refactor it out of the string interp